requirements_doc.txt

```diff
@@ -3,12 +3,12 @@ sphinx!=4.1.0
 https://github.com/numpy/numpydoc/archive/main.zip
 https://github.com/pydata/pydata-sphinx-theme/archive/cf8b113a496eb458587c0778a423e40c7c1aa22d.zip
 https://github.com/sphinx-gallery/sphinx-gallery/archive/master.zip
-sphinxcontrib-bibtex>=2.1.2
+git+https://github.com/mcmtroffaes/sphinxcontrib-bibtex.git
```

Comment:
`git+` is not as good as the `archive` or `zipball` variants of a direct `https` URL, because it actually does a full `git clone` IIRC

Reply:
Hmmm maybe we should just close since the CIs pass and wait for their releases

Reply:
Agreed, I think we don't have to rush on this